docs/install/windows-release-channels.md

```diff
@@ -0,0 +1,320 @@
+   - Upgrade to the Stable channel
+   - Preserve your configuration and data
+   - Restart the Netdata service
+
+</details>
+
+<br/>
+
+<details>
+<summary><strong>Switch to Nightly Channel</strong></summary><br/>
+
+1. **Download the Nightly MSI:**
+   - Go to: `https://github.com/netdata/netdata-nightlies/releases/latest/download/netdata-x64.msi`
+   - Save the file to your Downloads folder
+
+2. **Run the installer:**
+   - Double-click the downloaded `.msi` file
+   - Grant Administrator privileges when prompted
+   - Follow the installation wizard
+
+3. **Verify the installation:**
+   - The installer will automatically detect your existing installation
+   - Upgrade to the Nightly channel
+   - Preserve your configuration and data
+   - Restart the Netdata service
+
+</details>
+
+### Method 2: PowerShell Installation (Automated)
+
+<details>
+<summary><strong>Switch to Stable Channel via PowerShell</strong></summary><br/>
+
+```powershell
+# Run PowerShell as Administrator
+$ProgressPreference = 'SilentlyContinue'
+Invoke-WebRequest https://github.com/netdata/netdata/releases/latest/download/netdata-x64.msi -OutFile "$env:TEMP\netdata-x64.msi"
+msiexec /qn /i "$env:TEMP\netdata-x64.msi" REINSTALL=ALL
+```
+
+</details>
+
+<br/>
+
+<details>
+<summary><strong>Switch to Nightly Channel via PowerShell</strong></summary><br/>
+
+```powershell
+# Run PowerShell as Administrator
+$ProgressPreference = 'SilentlyContinue'
+Invoke-WebRequest https://github.com/netdata/netdata-nightlies/releases/latest/download/netdata-x64.msi -OutFile "$env:TEMP\netdata-x64.msi"
+msiexec /qn /i "$env:TEMP\netdata-x64.msi" REINSTALL=ALL
+```
+
+</details>
+
+### Method 3: PowerShell with Cloud Connection Preserved
+
+<details>
+<summary><strong>Switch Channels with Explicit Cloud Settings</strong></summary><br/>
+
+If you want to ensure your Netdata Cloud connection is maintained during the switch, you can provide your claim token and room IDs explicitly.
+
+**Get your current cloud settings:**
+
+```powershell
+# Check current cloud configuration
+Get-Content "C:\Program Files\Netdata\etc\netdata\cloud.d\cloud.conf"
+```
+
+Look for your `token` and `rooms` values.
+
+**Switch to Stable with Cloud settings:**
+
+```powershell
+# Run PowerShell as Administrator
+$ProgressPreference = 'SilentlyContinue'
+Invoke-WebRequest https://github.com/netdata/netdata/releases/latest/download/netdata-x64.msi -OutFile "$env:TEMP\netdata-x64.msi"
+msiexec /qn /i "$env:TEMP\netdata-x64.msi" TOKEN="<YOUR_TOKEN>" ROOMS="<YOUR_ROOMS>" REINSTALL=ALL
+```
+
+**Switch to Nightly with Cloud settings:**
+
+```powershell
+# Run PowerShell as Administrator
+$ProgressPreference = 'SilentlyContinue'
+Invoke-WebRequest https://github.com/netdata/netdata-nightlies/releases/latest/download/netdata-x64.msi -OutFile "$env:TEMP\netdata-x64.msi"
+msiexec /qn /i "$env:TEMP\netdata-x64.msi" TOKEN="<YOUR_TOKEN>" ROOMS="<YOUR_ROOMS>" REINSTALL=ALL
+```
```

Comment:
After we update our installer (https://github.com/netdata/netdata/pull/21090) it is not necessary to have `REINSTALL=ALL`, please, remove it.